apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx

```diff
@@ -54,10 +54,14 @@ Now, check if the installation of the VTEX Headless CMS plugin was successful by
 ![success-vtex-cms-sync](https://vtexhelp.vtexassets.com/assets/docs/src/faststore-vtex-cms___460365970fb39c94e300b9fbd497e17e.png)
 
 <Callout type="warning" emoji="⚠️">
-  If you use Windows as the operating system and find problems while installing
-  the plugin, please refer to our
+  <li>**Windows user:** if you find problems while installing
+  the plugin, please refer to the
   [Troubleshooting](/docs/headless-cms-troubleshooting/error-installing-headless-cms)
-  article.
+  article.</li> 
+  <br></br>
+  <li>**macOS user:** if you find problems, run `yarn config set ignore-engines true` to ignore the Node.js incompatibility version, then run `vtex plugins install cms` again. 
+  *Remember, you must have the latest Node.js version or a version higher than `15.0.0`.* </li> 
```

Comment:
This command is to ignore the node version incompatibility so I think here should be smt like:
*Remember, it is recommended to use the latest Node.js version or a version higher than `15.0.0`.*

Reply:
Thanks for the feedback, Bibi <3